.eslintrc.js

```diff
@@ -244,6 +244,7 @@ module.exports = {
         // This most often reports data that is defined after the component definition.
         // This is safe to do and helps readability of the demo code since the data is mostly irrelevant.
         '@typescript-eslint/no-use-before-define': 'off',
+        'react/prop-types': 'off',
```

Comment:
Why did you decided to disable this on the whole project?

Reply:
It's just for the demos. I don't think proptypes add anything valuable.
Currently, they were required for JS files, but not for TS and since we generate one from the other, it was tricky to make things right.